Lib/inspect.py

```diff
@@ -268,7 +268,7 @@ def iscode(object):
         co_kwonlyargcount   number of keyword only arguments (not including ** arg)
         co_lnotab           encoded mapping of line numbers to bytecode indices
         co_name             name with which this code object was defined
-        co_names            tuple of names of local variables
+        co_names            tuple of names of global variables
```

Comment:
The difference here is that "tuple of names of global variables" would be potentially misleading, since only those global variables used by the bytecode are included.  (IIUC)  This item is parallel to co_consts in that sense.